apps/site/components/Downloads/Release/ReleaseCodeBox.tsx

```diff
@@ -29,13 +31,16 @@ const ReleaseCodeBox: FC = () => {
   }, [release.versionWithPrefix, os, platform]);
 
   const codeLanguage = os === 'WIN' ? 'PowerShell' : 'Bash';
-
   return (
     <div className="mb-2 mt-6 flex flex-col gap-2">
+      {semVer.lt(release.versionWithPrefix, '18.0.0') && (
```

Comment:
where does ‘18.0.0’ come from? and why ?

Reply:
yeah, sadly it's hardcoded. i didn't see that this was encoded anywhere in metadata, but 18 is the oldest "supported" version as of today. This is definitely not ideal as it would need to be updated whenever new versions fall out of support.